### ISSUE TITLE
Pause screen

### ISSUE THREAD

**Issue by antila** [antila/ludum-dare-28#3]
Screen that pauses music and gameplay, with option to go back to main menu. Now it just goes to main menu when pushing ESC.

**Comment by ttymed** [comment]
Hey, I know this was a long time ago, but are you guys alright with me making this pause menu?

**Comment by nahkala** [comment]
Yes, please do! Any help with this project is greatly appreciated! We would love for this game to be a small completed freeware game some day. Unfortunately we haven't looked at the project for a while so we probably won't be of much help if you get stuck tough :(

**Comment by ttymed** [comment]
Ok, sweet.
I'll get to work on it then.
